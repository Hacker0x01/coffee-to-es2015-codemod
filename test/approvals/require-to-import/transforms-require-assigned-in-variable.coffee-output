var React = require("react/addons");
