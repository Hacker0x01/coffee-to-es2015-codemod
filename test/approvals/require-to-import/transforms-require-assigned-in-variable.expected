import React from "react/addons";
